src/components/Carousel/components/Controller/components/Item/Item.module.scss

```diff
@@ -0,0 +1,151 @@
+@use 'sass:math';
+@use '../../../../../../styles/utilities/variables' as *;
```

Comment:
cant these be relative paths?

Reply:
I think you meant to say absolute path, and I haven't been able to get absolute paths working on sass imports yet. We should revisit on another pr as it will likely require some vite config or storybook config changes